test/format/ops.c

```diff
@@ -1167,22 +1167,18 @@ ops(void *arg)
                      * Edge case: There is a case where we cannot detect a proper mirror mismatch.
                      * Say we truncated the tail end key range of all the mirrors from N to
                      * max_rows. This truncate happened before any thread added another non-mirrored
-                     * append/insert to a VLCS table and the data in that truncated key range was
-                     * sufficient to delete the pages at the end of the VLCS table. Then when a VLCS
-                     * non-mirrored insert happened, it appended the new item at key N instead of at
-                     * max_rows + 1.
-                     *
-                     * Then the next mirror check will detect a mismatch if there is an FLCS table
-                     * because the appended value does not match the FLCS truncated value. The
-                     * mismatch does not trigger with row-store tables because the row-store code to
-                     * get the next mirror key in format returns WT_NOTFOUND because all those
-                     * mirror keys are gone. But FLCS maintains all record numbers.
+                     * append/insert to a column store table and the data in that truncated key
+                     * range was sufficient to delete the pages at the end of the column store
+                     * table. Then when a column store non-mirrored insert happened, it appended the
+                     * new item at key N instead of at max_rows + 1. Then the next mirror check will
+                     * detect a mismatch from the row-store table because the appended value does
+                     * not match the truncated value.
                      *
                      * We want to test truncate at the end of the range as much as possible, so
                      * adjust the end range to max_rows - 1 only in the case where we are mirroring
                      * and have both a VLCS and FLCS table.
```

Comment:
The comment also needs to be corrected with `mirroring column store table` instead of `both VLCS and FLCS table`.

Reply:
done! [e17a054](https://github.com/wiredtiger/wiredtiger/pull/9523/commits/e17a054a2412208eb40c14b4e06aa081d38a130c)